docs/prerequisites.md

```diff
@@ -1,43 +1,60 @@
-Requirements (PC)
+Requirements
 ==
+
+- An Android 8+ device (to install the patched APK on)
+
+### If patching on PC:
+
 - ADB
 - x86/x86_64 host architecture
 - Zulu JDK 17
 
+### If patching on Android:
+
+- arm64-v8a host architecture
```

Comment:
I'm not sure if this is true for all patching methods. If not, I think there should be a notice that different patching tools have different requirements or it should just say the requirements for each tool. This is because theoretically, the user is meant to read and complete the steps in the prerequisites before even considering which patching tool to use. So we shouldn't have a user waste their time downloading a supported apk from apkmirror only to discover afterwards that none of their devices meet the requirements for any of the android patching methods.

Reply:
It seems appropriate to move the prerequisites to each patch method

Reply:
Also the PC methods? 

I think it makes sense to remove the PC method prerequisites and change the filename of `prerequisites.md` to `supplying-an-apk.md`, which all methods will reference.

Reply:
yeah it's better to integrate into one

Reply:
What do you mean?

Reply:
Remove duplicate `Common` (prerequisites) from each patch document and integrate them into `supplying-an-apk.md`
https://github.com/inotia00/revanced-documentation/blob/main/docs/rvx-manager.md#common
https://github.com/inotia00/revanced-documentation/blob/main/docs/rvx-builder.md#common